### ISSUE TITLE
Tool doesnt handle xattrs

### ISSUE THREAD

**Issue by brianredbeard** [arp242/elles#2]
I attempted to show the SELinux specific xattrs on a file:

```
❯ ./elles-v0.1.0-linux-amd64 -lZ
elles-v0.1.0-linux-amd64: unknown flag: "-lZ"
```

**Comment by arp242** [comment]
Yeah, it's one of the [TODO items](https://github.com/arp242/elles?tab=readme-ov-file#todo):

> There is no way to display file flags, ACLs, MAC labels, whiteouts, capabilities, or anything like that.

It's not so easy to do this well in a cross-platform way, especially since Windows is also a supported target. I'm not really sure what I would and wouldn't consider in-scope here either.
